lib/topological_inventory/core/service_plan_client.rb

```diff
@@ -0,0 +1,21 @@
+module TopologicalInventory
+  module Core
+    class ServicePlanClient
+      def build_payload(service_plan_name, service_offering_name, catalog_parameters)
+        {
+          "apiVersion" => "servicecatalog.k8s.io/v1beta1",
+          "kind"       => "ServiceInstance",
+          "metadata"   => {
+            "name"      => "#{service_offering_name}-#{SecureRandom.uuid}",
+            "namespace" => "default"
```

Comment:
@agrare The Service Portal can pass in the namespace as part of provider_control_parameters hash. Is this temporary?

Reply:
That's what I thought too but when I do that I get a 400 bad request